main.py

```diff
@@ -0,0 +1,113 @@
+        bot.register_next_step_handler(message, start_search0);
+
+
+def start_search0(message):
+    global nick
+    nick = message.text
+    if asdasd.check_name(nick):
+        asdasd.write_player(nick)
+        bot.send_message(message.from_user.id, "I'm glad to see you, "+ nick +'''\n/get_some_news — choose the country whose news you want
+        to receive,\n/help — Repeats this message,\n/stats - Get stats''')
+        bot.register_next_step_handler(message, start_search)
+    else:
+        bot.send_message(message.from_user.id, "This user is taken, please enter another one")
+        bot.register_next_step_handler(message, start_search0)
+    if message.text == "/help":
+        bot.send_message(message.from_user.id,
+                         "I'm glad to see you, "+ nick +'''\n/get_some_news — choose the country whose news you want to receive,
+                         \n/help — Repeats this message,\n/stats - Get stats''')
+        bot.register_next_step_handler(message, start_search);
+
+
+def start_search(message):
+    #theBoard = [' '] * 10
+    if message.text == "/get_some_news":
+        bot.send_message(message.from_user.id,
+                         "You want to recieve news according to  \n/category \n /keyword")
+        bot.register_next_step_handler(message, ch_country);
+    elif message.text == "/stats":
+        bot.send_message(message.from_user.id,
+                         "I've prepared for you some stats")
+        #asdasd.get_stats(Stat.name)
+        #bot.register_next_step_handler(message, start_search0);
+        stats_set = asdasd.get_stats(nick)
+        business_number_of_requests = str(int(stats_set[1]))
+        entertainment_number_of_requests = str(int(stats_set[2]))
+        bot.send_message(message.from_user.id,
+                         'Stats for ' + nick + ':\n\nbusiness_number_of_requests: ' + business_number_of_requests + '''\nentertainment_number_of_requests: 
+                         ''' + entertainment_number_of_requests)
+        bot.register_next_step_handler(message, start_search)
+    else:
+        bot.send_message(message.from_user.id, "I don't understand you, please type /help.")
+        bot.register_next_step_handler(message, get_text_messages);
+
+
+def ch_country(message):
+    if message.text == "/category":
+        bot.send_message(message.from_user.id,
+                         "Сhoose the category of preferable news \n\n/business\n/entertainment\n/general\n/health\n/science");
+        bot.register_next_step_handler(message, sho);
+    elif message.text == "/keyword":
+        bot.send_message(message.from_user.id,
+                         "I'm waiting for your word^^");
+        bot.register_next_step_handler(message, sho2);
+    else:
+        bot.send_message(message.from_user.id, "I don't understand you, please type /help.")
+        bot.register_next_step_handler(message, get_text_messages);
+
+
+def sho(message):
```

Comment:
Probably it's better to add docs to these functions (`sho`, `sho2` or to give them more informative names)